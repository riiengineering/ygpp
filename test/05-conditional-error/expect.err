test/05-conditional-error/input:8: error: Foo is defined.
