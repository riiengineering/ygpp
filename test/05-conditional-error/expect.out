Some text before the first conditional error.
Some text after the first #if.
